main.py

```diff
@@ -2,6 +2,41 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [("George", "Klooney"),
+              ("Kianu", "Reaves"),
+              ("Scarlett", "Keegan"),
+              ("Will", "Smith"),
+              ("Jaden", "Smith"),
+              ("Scarlett", "Johansson")]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for name, surname in actors:
+        Actor.objects.create(first_name=name, last_name=surname)
+
+    drama = Genre.objects.get(name="Dramma")
+    drama.name = "Drama"
+    drama.save()
```

Comment:
Try to do the same without a redundant variable in one line with the method `update()`. The same in other similar cases

Reply:
done